template.rb

```diff
@@ -168,10 +168,7 @@
     resource :checkout_session, only: :new
     resource :checkout_guest_session, only: :create
 
-    resource :checkout, only: :edit
-
-    # non-restful checkout stuff
-    patch '/checkout/update/:state', to: 'checkouts#update', as: :update_checkout
+    resource :checkout, only: [:edit, :update]
```

Comment:
This is the Rails convention for declaring a [singular resource](https://guides.rubyonrails.org/routing.html#singular-resources). As a singular resource, the paths that the route generates does not have the id param e.g. `/checkout` instead of `checkouts/:id`. However, also by Rails convention, the controller mapped to this route maintains a pluralized name (see `templates/app/controllers/checkouts_controller.rb`).

Reply:
Ah right, sorry!

Reply:
Hehe.. No worries!